setup.py

```diff
@@ -56,7 +56,7 @@ def _get_plugin_version() -> str:
     install_requires=[
         "dbt-spark>=1.8.0, <2.0",
         "dbt-core~=1.8.0",
-        "dbt-adapters~=1.2.0",
+        "dbt-adapters~=1.3.0",
```

Comment:
```suggestion
        "dbt-adapters~=1.2",
```

Can we do just this so we don't need to bump next time we do a minor release?

Reply:
Ok, but if you violate SemVer...